plugins/modules/aci_ip_sla_monitoring_policy.py

```diff
@@ -0,0 +1,320 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: aci_ip_sla_monitoring_policy
+short_description: Manage IP SLA Monitoring Policies (fv:IPSLAMonitoringPol)
+description:
+- Manage IP SLA Monitoring Policies used for L4-L7 Policy Based Redirection
+options:
```

Comment:
Please add option for:

- request data size (bytes)
- type of service
- operation timeout (ms)
- threshold (ms)
- traffic class